modules/administration-guide/examples/checluster-properties.adoc

```diff
@@ -11,7 +11,7 @@ airGapContainerRegistryOrganization: Optional repository name of an alternate co
 allowUserDefinedWorkspaceNamespaces: Deprecated. The value of this flag is ignored. Defines that a user is allowed to specify a Kubernetes namespace, or an OpenShift project, which differs from the default. It's NOT RECOMMENDED to set to `true` without OpenShift OAuth configured. The OpenShift infrastructure also uses this property.
 cheClusterRoles: A comma-separated list of ClusterRoles that will be assigned to Che ServiceAccount. Each role must have `app.kubernetes.io/part-of=che.eclipse.org` label. Be aware that the Che Operator has to already have all permissions in these ClusterRoles to grant them.
 cheDebug: Enables the debug mode for Che server. Defaults to `false`.
-cheFlavor: Deprecated. The value of this flag is ignored. Specifies a variation of the installation. The options are `che` for upstream Che installations, or `devspaces` for link\:https\://developers.redhat.com/products/codeready-workspaces/overview[Red Hat OpenShift Dev Spaces] installation. Override the default value only on necessary occasions.
+cheFlavor: Deprecated. The value of this flag is ignored. Specifies a variation of the installation. The options are  `che` for upstream Che installations or `devspaces` for Red Hat OpenShift Dev Spaces (formerly Red Hat CodeReady Workspaces) installation
```

Comment:
Don't commit this file as part of this pull request. The file is single-sourced from the code repository and the change belongs to a dedicated pull request.